code/src/main/java/com/onlinecontacttracing/messaging/LocalityResource.java

```diff
@@ -0,0 +1,73 @@
+package com.onlinecontacttracing.messaging;
+
+public enum LocalityResource implements HasEnglishTranslation, HasSpanishTranslation{
+  US("https://www.cdc.gov/coronavirus/2019-ncov/index.html", "Here's the national help link: ", "Aquí está el enlace de ayuda nacional: "),
```

Comment:
So, The big thing to note here is that everything is repeated except the state link value.That indicates to me that this can be factored out into a common variable rather than stored in each enum.

Reply:
Also, what led to this big enum structure like this? It feels slightly unwieldy to me. Though it's difficult with a big list. 

How did you come upon this structure in particular?

Reply:
It was something Grady and I had discussed when we were writing the DD, I can segment out a bit of it I think but I don't think I can completely eliminate all the enum values unless you've got any ideas?

Reply:
I agree with Ben that we don't want all of the duplicate data. For example, we know that "Here's your state's help link" doesn't change relative to the state. So that should just be a static variable used in the getEnglishTranslation method. The State Health links on the otherhand do varry by state, so they do need to be part of the enum construction.